docker/docker-compose.yml

```diff
@@ -1,16 +0,0 @@
-version: '3.7'
-
-services:
-  pgasync-postgres:
-    container_name: pgasync-postgres
-    image: postgres:11
-    environment:
-      - PGDATA=/database
-      - POSTGRES_PASSWORD=some_password
-      - TZ=America/New_York
-    volumes:
-      - .:/app
-      - ./database:/database
-      - ./docker-entrypoint-initdb.d:/docker-entrypoint-initdb.d
-    ports:
-      - "5432:5432"
```

Comment:
Why remove this?

Reply:
Good point, it's still listed in the readme. Will restore it and make sure it's updated to continue working.